src/browser/api_protocol/api_handlers/browser_view.ts

```diff
@@ -0,0 +1,47 @@
+import * as apiProtocolBase from './api_protocol_base';
+import { ActionSpecMap } from '../shapes';
+import { Channel } from '../../api/channel';
+import { Identity, APIMessage, ProviderIdentity } from '../../../shapes';
+import { AckFunc, NackFunc } from '../transport_strategy/ack';
+import * as browser_view from '../../api/browser_view';
+import { getBrowserViewByIdentity } from '../../core_state';
+
+const successAck = {
+    success: true
+};
+
+async function create (identity: Identity, message: APIMessage) {
+    const { payload } = message;
+    await browser_view.create(payload);
+    return successAck;
```

Comment:
also, we need to try/catch this browser_view.create

Reply:
Middleware catches and nacks with the proper message